packages/bits/demo/src/components/demo/wizard-v2/wizard-with-custom-icons/wizard-with-custom-icons-example.component.html

```diff
@@ -1,29 +1,39 @@
-<nui-wizard-horizontal [customIcons]="{selected: 'edit'}">
+<nui-wizard-horizontal>
 
     <nui-wizard-step-v2 i18n-label label="First step">
-        <span i18n>This is the first step of our wizard</span>.
+        <div class="p-5">
+            <span i18n>This is the first step of our wizard</span>.
+        </div>
         <ng-template wizardStepFooter>
             <ng-container *ngTemplateOutlet="wizardNext"></ng-container>
         </ng-template>
     </nui-wizard-step-v2>
 
-    <nui-wizard-step-v2 i18n-label label="Second step" [icon]="'email'">
-        <span i18n>This is the second step of our wizard</span>.
+    <!--
+        See how we use the 'stepIconsConfig' input here to override the global icons configuration
+        for this particular wizard step
+    -->
+    <nui-wizard-step-v2 i18n-label label="Second step" [stepIconsConfig]="secondStepIconConfig">
```

Comment:
Maybe we could rename the input `iconsConfig` since it's already in a context where the `step` part is redundant?

Reply:
The `iconsConfig` does not fit because the step component does not have any icons. I see now that the `stepIconsConfig`, however, does not fit either because of the same reason. Maybe `stateIconsConfig` will be a better choice because icons are directly bound to the step state, rather than to the state component itself.